docs/background_info/oid_tid_generator.md

```diff
@@ -110,13 +110,13 @@ In the QGIS Project the default value expression for `t_ili_tid` therefore is:
 | City_V1.TypeID | <code>'_' &#124;&#124; uuid('WithoutBraces')</code> |
 | not defined | <code>'_' &#124;&#124; uuid('WithoutBraces')</code> |
 
-### TID (OID) Manager
+### OID Manager
 
 Since the user has to be able to edit those values, they are provided in the GUI.
 
-Additionally to the [page implemented in the wizard](../../user_guide/import_workflow/#tid-oid-values) the configuration of the TIDs can be made on existing QGIS Projects.
+Additionally to the [page implemented in the wizard](../../user_guide/import_workflow/#tid-oid-values) the configuration of the OIDs can be made on existing QGIS Projects.
```

Comment:
How does those links work? They search for a chapter with that name using the `-` as a wildchar?

In that case the link should become `../../user_guide/import_workflow/#oid-values`

In other places as well... maybe there is a pipeline action that can check those internal links?

Reply:
Yes. You are right, it should be #oid-values The - is representing a space.

Good idea to check the links in a pipeline. I wonder if there is something existing for that :thinking:

Reply:
I think Adrien did something in the talking repo. Don't know if it can be used here